client/gutenberg/extensions/slideshow/save.js

```diff
@@ -11,5 +11,6 @@ export default ( { attributes: { align, autoplay, delay, effect, images }, class
 		delay={ delay }
 		effect={ effect }
 		images={ images }
+		forSave
```

Comment:
I think we can drop this now -- it's no longer used anywhere, is it?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30997/commits/d0ba0870f9241caf0a39db97d90372d1f5245eff.